sdk/digitaltwins/digital-twins-core/src/digitalTwinsClient.ts

```diff
@@ -154,12 +153,15 @@ export class DigitalTwinsClient {
     options: OperationOptions = {}
   ): Promise<DigitalTwinsGetByIdResponse> {
     const digitalTwinsGetByIdOptionalParams: DigitalTwinsGetByIdOptionalParams = options;
-    const span = this.createSpan(
+    const { span, updatedOptions } = createSpan(
       "DigitalTwinsClient-getDigitalTwin",
       digitalTwinsGetByIdOptionalParams
     );
     try {
-      return this.client.digitalTwins.getById(digitalTwinId, digitalTwinsGetByIdOptionalParams);
+      return this.client.digitalTwins.getById(
+        digitalTwinId,
+        operationOptionsToRequestOptionsBase(updatedOptions)
```

Comment:
remove the usage of this function everywhere

```suggestion
        updatedOptions
```

Reply:
Fixed